ui/src/app/login/components/login.tsx

```diff
@@ -15,24 +15,24 @@ const getToken = () => {
 
 const maybeLoggedIn = () => !!getToken();
 const logout = () => {
-    document.cookie = 'authorization=;';
+    document.cookie = 'authorization=;Max-Age=0';
     document.location.reload(true);
 };
 const login = (token: string) => {
-    document.cookie = 'authorization=' + token + ';';
+    document.cookie = 'authorization=' + token + ';SameSite=Strict';
     document.location.href = uiUrl('');
 };
 export const Login = () => (
     <Page title='Login' toolbar={{breadcrumbs: [{title: 'Login'}]}}>
         <div className='argo-container'>
             <p>
-                <i className='fa fa-info-circle' /> You appear to be logged {maybeLoggedIn() ? 'in' : 'out'}. It may not be necessary to login to use Argo, it depends on how it is
-                configured.
+                <i className='fa fa-info-circle' /> You appear to be <b>logged {maybeLoggedIn() ? 'in' : 'out'}</b>. It may not be necessary to login to use Argo, it depends on how
+                it is configured.
```

Comment:
A link to https://github.com/argoproj/argo/blob/master/docs/argo-server.md would be useful here

Reply:
yes - but not today :(